cmake-integration.el

```diff
@@ -92,7 +92,7 @@ project."
             (file-name-concat project-root-folder cmake-integration-build-dir)
           ;; If we do not have cmake-integration-build-dir set (it is
           ;; nil) throw an error asking the user to select a preset
-          (error "Please call `cmake-integration-cmake-configure-with-preset` first and select a \"configure preset\", or set \"cmake-integration-build-dir\" to be the build folder")
+          (error "Please call `cmake-integration-cmake-configure-with-preset' first and select a \"configure preset\", or set `cmake-integration-build-dir' to be the build folder")
```

Comment:
I have to get used to this convention on how to quote inside strings and documentation in elisp.
Thanks

Reply:
Yes, it's a bit weird... ;-)